tests/headers/class.hpp

```diff
@@ -2,6 +2,8 @@ class C {
     int a;
     // More than rust limits (32)
     char big_array[33];
+    char zero_length_array[0];
```

Comment:
I'm not sure this is legal C/C++? (the `zero_length_array` and `incomplete_array` fields would have the same address).

Could you remove the `zero_length_array` field from the test?